src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ## Unreleased
 
+* **Experimental Feature** Added an opt-in feature to aggregate any metric
+  measurements that were dropped due to reaching the max MetricPoints limit.
```

Comment:
a ref link to this might be helpful https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/metrics/customizing-the-sdk#changing-maximum-metricpoints-per-metricstream